### ISSUE TITLE
Gtk::Widget docs are incomplete

### ISSUE THREAD

**Issue by BlobCodes** [hugopl/gtk4.cr#4]
Currently, when generating the docs, the `Gtk::Widget` page ends like this:
![Bildschirmfoto vom 2022-01-03 15-01-06](https://user-images.githubusercontent.com/14961554/147939422-53d863cf-fa7c-48ed-bada-3300a9782e0e.png)

This is the related code:
```crystal
  #     </object>
  #   </child>
  # </object>
  # ```
  #
  # `GtkWidget` allows style information such as style classes to
  # be associated with widgets, using the custom `<style>` element:
  #
  # ```xml
```

So it seems like the `<style>` element is actually interpreted as an html style element.

**Comment by hugopl** [comment]
The doc feature is currently just a proof of concept, I just load the XML files that have the docs and use a xpath to fetch the contents, no processing is done yet and IIRC just the class descriptions are generated in the current POC.

I would like to have a way to put the docs on github pages and let them auto update on commits, but IMO deal with github actions is *really* a boring task, so it's not done.

**Comment by BlobCodes** [comment]
When I'm working with this library, I always use the auto-generated docs from libadwaita.cr.
Maybe you can get [_inspiration_](https://github.com/GeopJr/libadwaita.cr/blob/main/.github/workflows/docs.yml) from there.

Anyways having processing would be very cool (for example to show the widget previews).
But this issue shouldn't need any changes in gi-crystal as the doc generator fix has already been added to the crystal 1.4.0 milestone

**Comment by BlobCodes** [comment]
Fixed in https://github.com/crystal-lang/crystal/releases/tag/1.3.1
